src/sardana/taurus/qt/qtgui/extra_macroexecutor/macrobutton.py

```diff
@@ -147,6 +147,11 @@ def _doorStateChanged(self, state):
         if state not in [PyTango.DevState.ON, PyTango.DevState.ALARM] and not self.ui.button.isChecked():
             door_available = False
 
+        if state in [PyTango.DevState.RUNNING]:
+            self.setButtonText(self.macro_name + ": Click to abort")
```

Comment:
I do not like "<macroname>:click to abort" text. From a UI ergonomy point of view a button should not say "click". It is assumed. It should just describe the action to be done in imperative form (e.g. "abort <macroname>")